pkgs/tools/system/nvtop/default.nix

```diff
@@ -43,24 +50,20 @@ stdenv.mkDerivation rec {
   ++ optional (!nvidia) "-DNVIDIA_SUPPORT=OFF"
   ++ optional amd "-DLibdrm_INCLUDE_DIRS=${libdrm}/lib/stubs/libdrm.so.2"
   ;
-  nativeBuildInputs = [ cmake gtest ] ++ lib.optional nvidia addOpenGLRunpath;
-  buildInputs = with lib; [ ncurses ]
-    ++ optional nvidia cudatoolkit
-    ++ optional amd libdrm
-  ;
+
+  doCheck = true;
```

Comment:
Looks like a +9/-7 line diff with the only functional change being adding `udev` to buildInputs? IMHO, it's generally a good idea have functional changes and style changes in separate commits.